modules/src/terminal.c

```diff
@@ -140,17 +148,18 @@ void redraw(str_t* text_buf, const str_t* input_buf) {
     /* Window decorations */
 
     // background
-    snow_draw_rect(win->fb, 0, 0, win->width, win->height, 0x00353535);
+    snow_draw_rect(win->fb, 0, 0, win->width, win->height, clr.bg_color);
     // title bar
-    snow_draw_rect(win->fb, 0, 0, win->width, 20, 0x00222221);
-    snow_draw_border(win->fb, 0, 0, win->width, 20, 0x00000000);
-    snow_draw_string(win->fb, win->title, 4, 3, 0x00FFFFFF);
+    snow_draw_rect(win->fb, 0, 0, win->width, tb_height, clr.tb_color);
+    snow_draw_border(win->fb, 0, 0, win->width, tb_height, clr.tb_bcolor);
+    snow_draw_string(win->fb, win->title, tb_padding, tb_height / 3, clr.tb_tcolor);
     // border of the whole window
-    snow_draw_border(win->fb, 0, 0, win->width, win->height, 0x00555555);
+    snow_draw_border(win->fb, 0, 0, win->width, win->height, clr.w_bcolor);
```

Comment:
Replacing all these drawing calls by `snow_draw_window(win, &clr);` works for me, gives a grey terminal as usual.